tests/cli/v012/test_init_pandas.py

```diff
@@ -497,10 +497,7 @@ def test_cli_init_on_new_project_with_broken_excel_file_without_trying_again(
         "- Please check the file and try again or select a different data file."
         in stdout
     )
-    assert (
-        "- Error: Excel file format cannot be determined, you must specify an engine manually."
-        in stdout
-    ) or (
+    assert ("- Error: File is not a recognized excel file" in stdout) or (
```

Comment:
NB: Added this change and same for another test as this was the change I was seeing locally. Happy to reverse it out though if a local issue to resolve. :)

Reply:
No longer relevant as these tests have been reverted.